package.json

```diff
@@ -23,6 +23,7 @@
   },
   "homepage": "https://github.com/zooniverse/scribes-of-the-cairo-geniza",
   "dependencies": {
+    "counterpart": "^0.18.1",
```

Comment:
A thought: our eventual plan is that we'll implement the Translations system created by @eatyourgreens to handle Hebrew & Arabic labels.

When this happens, would we still need `counterpart`? I see that currently it's only used to determine the user's default language, as stored in Classifications.

Reply:
The translations API will give you back the translated text for a Panoptes resource, like a tutorial or a workflow. You'll probably still need a translation library like counterpart to manage translations for the text in your interface, like the navigation links or button labels.